views/audit/README.md

```diff
@@ -0,0 +1,119 @@
+        ```
+        gcloud alpha logging sinks create my-example-sink \ 
+        bigquery.googleapis.com/projects/my-project-id/datasets/auditlog_dataset \
+        --log-filter='protoPayload.metadata.@type="type.googleapis.com/google.cloud.audit.BigQueryAuditMetadata" AND ( (protoPayload.metadata.jobChange.job.jobConfig.queryConfig.statementType="SCRIPT" AND protoPayload.metadata.jobChange.job.jobStatus.jobState="DONE" ) OR ( protoPayload.metadata.jobChange.job.jobStats.parentJobName!="" AND protoPayload.metadata.jobChange.job.jobStatus.jobState="DONE") OR protoPayload.metadata.tableDataChange.reason="QUERY")' \ 
+        --use-partitioned-tables
+        ``` 
+        Note: gcloud **alpha** is needed in order to use the parameter `--use-partitioned-tables` 
+    *   [Cloud Console Logs Viewer](https://cloud.google.com/logging/docs/export/configure_export_v2#dest-create)
+        Use this filter:
+        #### protoPayload.metadata.@type="type.googleapis.com/google.cloud.audit.BigQueryAuditMetadata" AND ( (protoPayload.metadata.jobChange.job.jobConfig.queryConfig.statementType="SCRIPT" AND protoPayload.metadata.jobChange.job.jobStatus.jobState="DONE" ) OR ( protoPayload.metadata.jobChange.job.jobStats.parentJobName!="" AND protoPayload.metadata.jobChange.job.jobStatus.jobState="DONE") OR protoPayload.metadata.tableDataChange.reason!="" OR protoPayload.metadata.tableDataRead.reason!=""  OR protoPayload.metadata.tableDeletion.reason!="" )
+        *   [Partitioning](https://cloud.google.com/logging/docs/export/bigquery#partition-tables)
+            is not required, but it is strongly recommended to select it for your BigQuery destination
+            
+    Note: You can create a log sink at the folder, billing account, or organization level using an 
+    [aggregated sink](https://cloud.google.com/logging/docs/export/aggregated_sinks#creating_an_aggregated_sink).
+1.  The log sink will immediately create the BigQuery dataset but the table will
+    be created once you run a BigQuery job post log sink creation.
+1.  To use the SELECT statement in
+    [big_query_elt_audit_log_v2.sql](/views/audit/big_query_elt_audit_log_v2.sql), change
+    all occurrences of
+    `project_id.dataset_id.cloudaudit_googleapis_com_data_access` to be the full
+    table path you created in step 1.
+    *   `sed
+        's/project_id.dataset_id.cloudaudit_googleapis_com_data_access/YOUR_PROJECT.YOUR_DATASET.YOUR_TABLE/'
+        big_query_elt_audit_log_v2.sql`
+1.  From here, you can do further analysis in BigQuery by querying the view, or
+    you can connect it to a BI tool such as DataStudio as a data source and
+    build dashboards.
+    
+#### Usage Examples
+Change all occurrences of `YOUR_VIEW` to the full path to the view. 
+
+* Run this query to see job name, query, total number of billed bytes, job creation time, job start time, job end time, job runtime, and count of inserted rows and deleted rows for DML queries in a script.
```

Comment:
Better descriptions needed for these usage examples. Don't list out columns being read in the description, but rather explain concisely what this is useful for. 

e.g. Execute the following query to list out all scripts which modified data in tables.

Reply:
Focus on explaining the WHERE filters you're using as well as the ORDER BY